app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :content, presence: true, length: { maximum: 60 }
+  validates :user_id, presence: true
```

Comment:
We already have the association established in lines 2-3. We don't need anymore to validate presence of user_id and post_id. :)